src/app/bitstream-page/clarin-license-agreement-page/clarin-license-agreement-page.component.ts

```diff
@@ -162,12 +162,7 @@ export class ClarinLicenseAgreementPageComponent implements OnInit {
     this.loadUserRegistrationAndUserMetadata();
 
     // Load the Seznam dataset license content
-    this.item$.subscribe((item$) => {
-        console.log(this.item$.value.metadata?.['dc.rights']?.[0]?.value);
-        if (this.item$.value.metadata?.['dc.rights']?.[0]?.value === this.LICENSE_NAME_SEZNAM) {
-          this.loadLicenseContentSeznam();
-        }
-    });
+    this.loadSeznamDatasetLicenseContent();
```

Comment:
We have two methods with almost the same name doing different things:
`loadSeznamDatasetLicenseContent` and `loadLicenseContentSeznam` - this is not right, it should be clear what is the method doing.